src/components/landing-page/LandingPage.jsx

```diff
@@ -3,17 +3,49 @@ import PropTypes from "prop-types";
 import i18n from "@dhis2/d2-i18n";
 import Paper from "@material-ui/core/Paper";
 import FontIcon from "material-ui/FontIcon";
-import MenuItem from "@material-ui/core/MenuItem";
-import MenuList from "@material-ui/core/MenuList";
 import { withStyles } from "@material-ui/core/styles";
 import { Link } from "react-router-dom";
+import Grid from "@material-ui/core/Grid";
+import ListItem from "@material-ui/core/ListItem";
+import Typography from "@material-ui/core/Typography";
 
 const styles = () => ({
     root: {
         display: "flex",
+        flexGrow: 1,
+        justifyContent: "center",
+    },
+    listItem: {
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        textDecoration: "none",
+        "&:hover": {
+            backgroundColor: "#f9f9f9",
+        },
+        cursor: "pointer",
+        padding: 50,
+    },
+    container: {
+        width: "90%",
+        padding: 10,
+    },
+    item: {
+        padding: 20,
+    },
+    title: {
+        marginLeft: 10,
+    },
+    icons: {
+        fontSize: "70px !important",
+        marginRight: 10,
```

Comment:
Depending on the zoom-level, elements in the grid may have different heights. Is this a problem? (or can it be solved?)

![image](https://user-images.githubusercontent.com/24643/60260308-85b3f780-98d9-11e9-8b09-07f7a5bb7234.png)